plug-ins/dns_spoof/dns_spoof.c

```diff
@@ -503,9 +521,14 @@ static int prepare_dns_reply(u_char *data, const char *name, int type, int *dns_
    struct rr_entry *rr;
    bool is_negative;
    int len;
+   u_int32 ttl, ttl_bige;
    u_char *answer, *p;
    char tmp[MAX_ASCII_ADDR_LEN];
 
+   /* set TTL to 1 hour by default or in case something goes wrong */
+   ttl = 3600;
+   ttl_bige = 0x10e00000; /* big endian for network stuff */
```

Comment:
is this equivalent to
```ttl_bige = htonl(ttl);```

if yes I would maybe prefer the htonl, less efficiency but more readability....